src/main/java/com/sfjs/gql/resolvers/Authorization.java

```diff
@@ -0,0 +1,36 @@
+package com.sfjs.gql.resolvers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.graphql.data.method.annotation.Argument;
+import org.springframework.graphql.data.method.annotation.MutationMapping;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import com.sfjs.gql.schema.Result;
+import com.sfjs.security.AuthorizationService;
+
+@RestController
+@EnableWebMvc
```

Comment:
Out of curiosity, are these annotations Spring needs even though we use Graphql?

Reply:
We have both GQL and REST API's.  But this class probably doesn't need these particular annotations since it is for GQL only.

Reply:
Cool, thanks for clarifying